pom.xml

```diff
@@ -224,12 +224,12 @@
 
     <!-- Google Libraries -->
     <google.cloud.libraries.version>26.40.0</google.cloud.libraries.version>
-    <google.auth.version>1.23.0</google.auth.version>
-    <gson.version>2.10.1</gson.version>
-    <guava.version>33.1.0-jre</guava.version>
-    <google.http-client.version>1.44.1</google.http-client.version>
-    <protobuf.version>3.25.2</protobuf.version>
-    <grpc.version>1.61.1</grpc.version>
+    <protobuf.version>3.25.2</protobuf.version> <!-- only getting used in proto compilation plugin -->
+    <grpc.version>1.61.1</grpc.version>  <!-- only getting used in proto compilation plugin -->
```

Comment:
Is there a way to extract these 2 versions from the BOM?

Reply:
Imported `grpc-bom` and `protobuf-pom` into the project. 

In this way, the version during compilation and runtime should be the same. However, while updating `google-bom,` we need to make sure that the `grpc` and `protobuf ` versions is supported or not.